README.md

```diff
@@ -43,6 +43,36 @@ python nutrition_labels/grant_tagger.py --training_data_file data/processed/trai
 
 where `vectorizer_type` can be 'count', 'tfidf' or 'bert', `model_type` can be 'naive_bayes', 'SVM' and 'log_reg', and `bert_type` (if using bert) can be 'bert' and 'scibert'.
 
+## finding_most_common_cohort.py / finding_most_common_references.py 
+
+To find the most common health data sets first run:
+
+```
+python nutrition_labels/finding_most_common_cohort.py
```

Comment:
i think these might be the wrong way around? This seems like the explanation of "finding_most_common_references.py"?

Reply:
you're right